junebug/stores.py

```diff
@@ -0,0 +1,59 @@
+from twisted.internet.defer import inlineCallbacks, returnValue
+from vumi.message import TransportUserMessage
+
+
+class BaseStore(object):
+    '''
+    .. class:: junebug.stores.BaseStore(redis, ttl)
```

Comment:
I think autodoc doesn't need the `.. class:: ...` -- it can figure that out from the fact that this is a class.